variables.yml

```diff
@@ -290,7 +290,7 @@ networks:
       collator_map_bond: 100
       max_candidates: 36
       round_blocks: 600
```

Comment:
```suggestion
      round_blocks: 1200
```

Reply:
ahhhhhh haha I double-checked everything in the configs but the most important thing 🤦‍♀️